nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/CommentsMigrator.java

```diff
@@ -215,22 +215,22 @@ protected void migrateSessionPropertyToSecured(CoreSession session, MigrationCon
 
             session.removeDocuments(documentsToRemove);
 
-            reportProgress("Done Migrating from Property to Secured", totalOfComments, totalOfComments);
+            reportProgress("Done Migration from Property to Secured", totalOfComments, totalOfComments);
```

Comment:
Not sure about this change.

Reply:
this was a warning of my spell checking IDE plugin, and i changed it. if you want i can rollback this part

Reply:
Probably because your dictionary doesn't have this word, it can happen. Yes, I'm in favor of not changing it and adding the word to the dictionary to remove the warning.

Reply:
The first one seems more correct. If spell checking is not happy maybe you can say
> "Migrated from Property to Secured"